### ISSUE TITLE
Hub doesn't release a node after a GRID_TIMEOUT

### ISSUE THREAD

**Issue by FzmTYustas** [SeleniumHQ/selenium#6559]
## Meta
OS: Ubuntu 14.04.3 LTS with latest docker images: Hub + NodeFirefoxDebug
Selenium Version: 3.14.0
Browser: Firefox
Browser Version:  62.0.3

## Environment
- 4 selenium/node-firefox-debug images + 1 selenium/hub images (run using docker-compose)
- GRID_TIMEOUT=30

Hub config:
![grid_config](https://user-images.githubusercontent.com/33956805/47214061-7de4a080-d3a5-11e8-8d46-51c6b24400f9.png)

## Steps
1. Run in a PyCharm:
```
from selenium import webdriver
from selenium.webdriver.common.desired_capabilities import DesiredCapabilities
from time import sleep

firefox = webdriver.Remote(
          command_executor='http://localhost:4444/wd/hub',
          desired_capabilities=DesiredCapabilities.FIREFOX)

firefox.get('https://www.google.com')

sleep(120)
```
2. Stop execution of the script after ~10 seconds.
3. Wait for ~50-60 seconds.

## Result
Node is still busy in a grid console.

## Expected
Node should be released after GRID_TIMEOUT seconds.

**Comment by FzmTYustas** [comment]
Can this be due to the fact that the node's TIMEOUT value was not updated with the hub's one?
![node_config](https://user-images.githubusercontent.com/33956805/47214689-72927480-d3a7-11e8-9b2d-1cc3241c2dd5.png)
And seems this is another bug.

**Comment by diemol** [comment]
Duplicate of #5418

**Comment by FzmTYustas** [comment]
Hi @diemol! Is the fix from #5418 suitable for GRID_TIMEOUT? I wasn't talking about browser_timeout.

**Comment by diemol** [comment]
Right, this applies to browserTimeout, let me check this and I'll report back.
